lesson07.blog-api-server/index.html

```diff
@@ -1,16 +1,69 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-    <title>Blog API Server</title>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-</head>
+    <title>Title</title>
+    <style>
 
+    </style>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
 <body>
-    <header>
-        <h1>Blog API Server</h1>
-    </header>
+<div class="container">
+    <div class="row">
+        <div class="col-sm btns">
+            <h1>JSON REQUEST SYSTEM </h1>
+            <button type = "button" class="btn btn-info" id="postsBtn"  value="posts">Show Posts</button>
+            <button  type = "button" class="btn btn-info" id ="photosBtn" value ="photos" >Show Photos</button>
+            <button type = "button" class="btn btn-info" id="commentsBtn" value ="comments">Show Comments</button>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm ">
+            <input type="text"  class="form-control inputParams" placeholder="User ID" id ="userIdParam" >
+            <input type="text"  class="form-control inputParams" placeholder="ID" id ="idParam" >
+            <input type="text"  class="form-control inputParams " placeholder="Sort By" id ="sortParam" >
+        </div>
+    </div>
+</div>
+
+<script src = "https://cdn.jsdelivr.net/npm/lodash@4.17.11/lodash.min.js"></script>
+
+<script>
+    'use strict'
+    const SERVER_URL = 'http://localhost:8080/';
```

Comment:
nice.
Note you can also use relative address in this case (do no need for the server address)